OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -156,7 +160,8 @@ public void ResolveOrder(Actor self, Order order)
 
 		bool IsCursorBlocked()
 		{
-			return ((deployState == DeployState.Deployed) && !Info.CanUndeploy) || (!IsValidTerrain(self.Location) && (deployState != DeployState.Deployed));
+			return !IsTraitPaused
```

Comment:
This line is a monster.  Can you please split this into multiple if statements to improve readability?

Reply:
Split the ITP check to it's own if and removed the canUndeploy query, since it was unnecessary anymore.